src/select/select.js

```diff
@@ -115,10 +115,16 @@ class Select extends React.Component<PropsT, StatelessStateT> {
     type: ChangeActionT,
     id?: string = '',
     label?: LabelT,
+    disabled?: boolean,
   ) => {
     const multiple = this.isMultiple();
     const selected = this.state.selectedOptions.find(tag => tag.id === id);
     let selectedOptions;
+
+    if (disabled) {
+      return;
+    }
```

Comment:
Not sure what the latest decision is on the event handling during disabled state, but added this which early return to stop changes from occurring.